defaults/main.yml

```diff
@@ -19,7 +19,7 @@ cartridge_multiversion: false
 instance_start_timeout: 60
 instance_discover_buckets_timeout: 60
 
-cartridge_install_dir: /usr/share/tarantool
+cartridge_app_install_dir: /usr/share/tarantool
 cartridge_instances_dir: /usr/share/tarantool
```

Comment:
What about to rename this variable? cartridge_instances_app_dir?

Reply:
I like `cartridge_instances_dir` name. It's simple and clear.